src/utils/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useState } from 'react';
+import { ItemProps, Todo } from '../../types/Todo';
+
+export const TodoItem: React.FC<ItemProps> = ({
+  todo,
+  handleDeleteTodo,
+  handleTodoStatusChange,
+  loadingTodoIds,
+  updateTodo,
+}) => {
+  const [editedTodo, setEditedTodo] = useState<string>('');
+  const [isDoubleClicked, setIsDoubleClicked] = useState<number | null>(null);
```

Comment:
it can be just a boolean value

Reply:
<img width="608" alt="image" src="https://github.com/mate-academy/react_todo-app-with-api/assets/70220667/66bbb529-0a1c-4409-a83a-a32291354a80">
still not fixed